README.md

```diff
@@ -163,6 +163,25 @@ helm delete ols-release --namespace openshift-lightspeed
 Chart customization is available using the [Values](helm/values.yaml) file.
 
 
+## Project structure
+
+1. REST API handlers
+1. Configuration loader
+1. LLM loader
+1. Doc retriever from vector storage
+1. Question validator
+1. Docs summarizer
```

Comment:
we need to come up with a better name for this component.  Right now it's really more of a "generic question answerer" (it uses RAG content when available, but if not it just sends the user question on to the LLM as is)